src/app/lang/ko.js

```diff
@@ -50,6 +50,9 @@ export default {
     no_custom_word_column:
       "사용자 지정 단어 열이 지정되지 않았습니다! 기본값은",
     no_custom_count_column: "사용자 지정 수 열이 지정되지 않았습니다! 기본값은",
+    existing_project_warning: "이미 프로젝트가 시작되었습니다!",
```

Comment:
This used Google translate. I assume a potentially bad translation is better than no translation?

Reply:
After the standup, the current workflow we'll want to have in the future is to only add translations for languages we are confident in, and then use English as a fallback.

I'll leave the Korean translation for now if @yehee gets an opportunity to review this, going forward I'll work with English only